pkg/arvo/sys/hoon.hoon

```diff
@@ -13833,8 +13833,8 @@
         |=(a=axis [%& a])
       ;~  pose
         ;~(pfix lus dim:ag)
-        ;~(pfix pam (cook |=(a=@ ?:(=(0 a) 0 (mul 2 +($(a (dec a)))))) dim:ag))
-        ;~(pfix bar (cook |=(a=@ ?:(=(0 a) 1 +((mul 2 $(a (dec a)))))) dim:ag))
+        ;~(pfix pam (cook |=(a=@ (sub (lsh [0 +(a)] 1) 2)) dim:ag))
+        ;~(pfix bar (cook |=(a=@ (sub (lsh [0 +(a)] 1) 1)) dim:ag))
```

Comment:
These could be written `(sub (bex +(a)) 2)` and `(sub (bex +(a)) 1)`.

Reply:
oh yeah good point